### DISCUSSION TITLE
delete overlay not working anymore

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#539] [Q&A] [answered]
B5:
Can you please check delete overlay is not working anymore.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

I tested with menu "Routing | Import" a gpx as overlay.
Then menu "Routing | Clear | Overlay" and it removes it.

What overlay do you mean or what steps to reproduce it?

**Comment by DolomitesTom** [comment, marked as answer]
sorry my fault, it was show route on map, topic closed
